### PR TITLE
Adding sort to releases route

### PR DESCRIPTION
**Pull request by andrehp**

### CODE REVIEW

**Comment by Henrod** [comment]
Please, fix the tests. Also, change the version on Chart.yaml

File: models/scheduler.go [hunk 1]
```diff
@@ -356,7 +356,7 @@ func ListSchedulerReleases(db interfaces.DB, schedulerName string) (
 ) {
 	_, err = db.Query(
 		&versions,
-		"SELECT version, created_at FROM scheduler_versions WHERE name = ?",
+		"SELECT version, created_at FROM scheduler_versions WHERE name = ? ORDER BY version",
```
Comment (Henrod) [inline models/scheduler.go]: The order by should be `ORDER BY created_at ASC` because version is a string

Comment (cscatolini) [reply models/scheduler.go]: we can order by strings as well

Comment (Henrod) [reply models/scheduler.go]: It wouldn't work for this case:
```
v10.0
v100.0
v11.0

Comment (cscatolini) [reply models/scheduler.go]: you're right :(

Comment (andrehp) [reply models/scheduler.go]: Agreed, will fix this

**Comment by andrehp** [comment]
Done

**Pull request merged**